fplus-lib/src/external_services/github.rs

```diff
@@ -156,12 +151,16 @@ impl GithubWrapper {
             }))
             .build()
             .expect("Could not create Octocrab instance");
-        let iod: InstallationId = installation_id.into();
-        let installation = octocrab.installation(iod);
+        if let Some(installation_id) = installation_id {
+            let installation_id: u64 = installation_id
+                .try_into()
+                .expect("Installation Id sucessfully parsed to u64");
+            octocrab.installation(installation_id.into());
+        }
```

Comment:
```suggestion
        let octocrab = if let Some(installation_id) = installation_id {
            let installation_id: u64 = installation_id
                .try_into()
                .expect("Installation Id sucessfully parsed to u64");
            octocrab.installation(installation_id.into())
        } else {
            octocrab
        }
```

There's a bug here. `octocrab.installation` doesn't mutate the `octocrab` - instead it returns a new instance with the installation set. So just `octocrab.installation(installation_id.into());` does nothing, we need to assign it to something. Otherwise current code will always create gh wrapper with octocrab with no installation added.

Personally I'd even change the `if let` into `installation_id.map_or`.

https://docs.rs/octocrab/latest/octocrab/struct.Octocrab.html#method.installation